docs/source/science/ATL08/ATL08.ipynb

```diff
@@ -0,0 +1,1389 @@
+{
```

Comment:
<p>This s3_url is used to access the data directly right? Might be good to explain that this cell pertains to the direct access approach</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/362/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>